client/lib/user/user.js

```diff
@@ -238,7 +238,28 @@ User.prototype.set = function( attributes ) {
 	return changed;
 };
 
+User.prototype.changeUser = function( username, password, callback ) {
+	if ( config.isEnabled( 'support-user' ) ) {
+		wpcom.changeUser( username, password, function( error ) {
+			if ( ! error ) {
+				this.fetch();
+			}
+			callback( error );
+		}.bind( this ) );
+	}
+};
+
+User.prototype.restoreUser = function() {
+	if ( config.isEnabled( 'support-user' ) ) {
+		wpcom.restoreUser();
+
+		this.fetch();
+	}
+}
+
 /**
  * Expose `User`
  */
 module.exports = User;
+
+require( './dev-support-user' );
```

Comment:
I'm not a fan of the require here. Is it possible to move this to a more appropriate boot section?